content/en/docs/concepts/security/hardening-guide/authentication-mechanisms.md

```diff
@@ -0,0 +1,83 @@
+
+When using TokenRequest tokens for service authentication, it is recommended to implement a short lifespan to reduce the impact of compromised tokens.
+
+## OpenID Connect Token Authentication {#openid-connect-token-authentication}
+
+Kubernetes supports integrating external authentication services with the Kubernetes API using [OpenID Connect (OIDC)](/docs/reference/access-authn-authz/authentication/#openid-connect-tokens). There is a wide variety of software that can be used to integrate Kubernetes with an identity provider. However, when using OIDC authentication for Kubernetes, it is important to consider the following hardening measures:
+
+- The software installed in the cluster to support OIDC authentication should be isolated from general workloads as it will run with high privileges.
+- Some Kubernetes managed services are limited in the OIDC providers that can be used.
+- As with TokenRequest tokens, OIDC tokens should have a short lifespan to reduce the impact of compromised tokens.
+
+## Webhook Token Authentication {#webhook-token-authentication}
+
+[Webhook token authentication](/docs/reference/access-authn-authz/authentication/#webhook-token-authentication) is another option for integrating external authentication providers into Kubernetes. This mechanism allows for an authentication service, either running inside the cluster or externally, to be contacted for an authentication decision over a webhook. It is important to note that the suitability of this mechanism will likely depend on the software used for the authentication service, and there are some Kubernetes-specific considerations to take into account.
+
+To configure Webhook authentication, access to control plane server filesystems is required. This means that it will not be possible with Managed Kubernetes unless the provider specifically makes it available. Additionally, any software installed in the cluster to support this access should be isolated from general workloads, as it will run with high privileges.
+
+## Authenticating Proxy {#authenticating-proxy}
+
+Another option for integrating external authentication systems into Kubernetes is to use an [authenticating proxy](/docs/reference/access-authn-authz/authentication/#authenticating-proxy). With this mechanism, Kubernetes expects to receive requests from the proxy with specific header values set, indicating the username and group memberships to assign for authorization purposes. It is important to note that there are specific considerations to take into account when using this mechanism.
+
+Firstly, TLS must be used between the proxy and Kubernetes API server to mitigate the risk of traffic interception or sniffing attacks. This ensures that the communication between the proxy and Kubernetes API server is secure.
```

Comment:
Any TLS version? Maybe we should be more specific.

Reply:
So here I was more recommending the use of TLS as against clear text connections, as opposed to getting into the details of TLS config. I can qualify it with "securely configured TLS" to indicate that they should do that well :)